app/presenters/content_item_presenter.rb

```diff
@@ -8,13 +8,20 @@ def initialize(content_item)
     @format = content_item["format"]
     @locale = content_item["locale"] || "en"
     @phase = content_item["phase"]
-    @links = content_item["links"] || {}
   end
 
   def available_translations
     sorted_locales(links["available_translations"])
   end
 
+  def links
```

Comment:
Is there any reason we need a method to do this? It feels like it might be easier to put...

```
@links = content_item["links"] || {}
@details ||= content_item["details"] || {}
```

...in the `initialize` method of this class, and include `attr_reader :links, :details` if they need to be accessed via methods.